Cargo.toml

```diff
@@ -30,6 +30,7 @@ libipld = { version = "0.12.0", default-features = false, features = ["dag-cbor"
 multihash = { version = "0.14.0", default-features = false, features = ["blake3"] }
 ipfs-embed = { version = "0.22.4" }
 anyhow = "1.0.41"
+thiserror = "1.0.30"
```

Comment:
Is this used somewhere or we can remove thiserror atm?

Reply:
no I will remove it.